examples/core/cubemap2/app.js

```diff
@@ -0,0 +1,143 @@
+import GL from '@luma.gl/constants';
+import {AnimationLoop, TextureCube, Cube, setParameters} from 'luma.gl';
+import {Matrix4, radians} from 'math.gl';
+
+const INFO_HTML = `
+<p>
+A <code>cubemapped</code> prism within a larger cubemapped cube
```

Comment:
This text needs to distinguish more from the existing example, to explain what this is showing.